Topic0Ejercicio3/src/buildingFactory.java

```diff
@@ -0,0 +1,12 @@
+
+public class buildingFactory {
```

Comment:
Class names in mixed case with the first letter of each internal word capitalized